.travis.yml

```diff
@@ -47,13 +50,39 @@ deploy:
   secret_access_key: "$AWS_SECRET_ACCESS_KEY_QA"
   on:
     branch: qa
+- provider: lambda
+  function_name: SierraHoldingParser-production
+  description: A function that parses holding records from the Sierra API
+  region: us-east-1
+  role: arn:aws:iam::946183545209:role/lambda-full-access
+  runtime: ruby2.7
+  timeout: 60
+  module_name: app
+  handler_name: handle_event
+  event: arn:aws:kinesis:us-east-1:946183545209:stream/SierraHoldingParser-production
+  vpc:
```

Comment:
This actually doesn't need to be in a VPC as it doesn't interact with anything else in there. I'm not sure if we consider it best practice to place functions in the VPC anyway though

Reply:
I tend to lean towards putting it in a VPC 1. to prevent anything outside the vpc from talking to it and 2. as a default practice, but in the present case it's probably unnecessary.

Reply:
okay, yeah, I'm actually curious to discuss this at retro with everyone because I'm like 50/50 on what standard practice should be. If you want to put the qa function in a vpc the config is:

Subnets:
- subnet-21a3b244
- subnet-f35de0a9

SG: sg-aa74f1db